app/views/devise/registrations/new.html.erb

```diff
@@ -1,31 +1,62 @@
-<h2>Sign up</h2>
 
-<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: :false }) do |f| %>
-  <%= f.error_notification %>
+<section class="vh-100%" style="background-color: #9A616D;">
+  <div class="container py-5 h-90">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col col-xl-10">
+        <div class="card" style="border-radius: 1rem;">
+          <div class="row g-0">
+            <div class="col-md-6 col-lg-6 d-none d-md-block">
+              <img src="https://images.pexels.com/photos/1074531/pexels-photo-1074531.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
+                alt="login form" class="img-fluid h-100" style="border-radius: 1rem 0 0 1rem;"/ >
+            </div>
+            <div class="col-md-3 col-lg-5 d-flex align-items-center">
+              <div class="card-body p-4 p-lg-5 text-black">
 
-  <div class="form-inputs">
-  <%= f.input   :first_name,
-                required: true,
-                autofocus: true%>
-  <%= f.input   :last_name,
-                required: true,
-                autofocus: true%>
-    <%= f.input :email,
-                required: true,
-                autofocus: true,
-                input_html: { autocomplete: "email" }%>
-    <%= f.input :password,
-                required: true,
-                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
-                input_html: { autocomplete: "new-password" } %>
-    <%= f.input :password_confirmation,
-                required: true,
-                input_html: { autocomplete: "new-password" } %>
-  </div>
+                <form>
+                  <div class="d-flex align-items-center mb-3 pb-1">
+                    <%= image_tag "logo.png", width:"130px" %>
+                    <span class="h1 fw-bold mb-0">Scedulazy</span>
```

Comment:
I would also say to use either only "Schedulazy" in the logo's font or only use the logo, not both